x-pack/solutions/security/plugins/security_solution/jest.integration.config.js

```diff
@@ -9,4 +9,7 @@ module.exports = {
   preset: '@kbn/test/jest_integration',
   rootDir: '../../../../..',
   roots: ['<rootDir>/x-pack/solutions/security/plugins/security_solution'],
+  globals: {
+    Uint8Array: Uint8Array,
+  },
```

Comment:
See https://github.com/jestjs/jest/issues/4422

Without setting this global [this condition](https://github.com/cthackers/adm-zip/blob/master/zipEntry.js#L24) fails because `Uint8Array` types are different, which in turn fails the tests

I've attempted to set once in the root jest config but ended up with a lot of unrelated tests failing, so I opted for setting it on specific configs instead